docs/config.json

```diff
@@ -19,6 +18,8 @@
       "icon": "wrench",
       "title": "Setup",
       "entries": [
+        { "title": "Configuration", "slug": "/setup/configuration/" },
+        { "title": "Installation", "slug": "/setup/installation/" },
```

Comment:
nit: probably installation should go first, as usually folks need to install things and then configure them